gensim/matutils.py

```diff
@@ -382,6 +383,110 @@ def cossim(vec1, vec2):
     return result
 
 
+def isbow(vec):
+    """
+    Checks if vector passed is in bag of words representation or not.
+    """
+    if scipy.sparse.issparse(vec):
```

Comment:
I don't think `isbow` should return `True` for scipy.sparse inputs. It's a conceptual mismatch.

There already exists a check for scipy.sparse inputs: `scipy.sparse.issparse()`.

Or what is the purpose of "isbow()", conceptually?